### PR TITLE
version tick (and minor api tweak)

### PR DESCRIPTION
**Pull request by ethn**

### CODE REVIEW

File: card/lib/card/set/event.rb [hunk 1]
```diff
@@ -59,11 +59,11 @@ def callback_name stage, after_subcards=false
 
       def define_event event, opts, &final
         simple_method_name = "#{event}_without_callbacks"
-        define_simple_method simple_method_name, &final
+        define_simple_method event, simple_method_name, &final
         define_event_method event, simple_method_name, opts
       end
 
-      def define_simple_method method_name, &method
+      def define_simple_method _event, method_name, &method
```
Comment (ethn) [inline card/lib/card/set/event.rb]: needed for new_relic mod

**Comment by xithan** [comment]
maybe include #152 in this version tick?

**Comment by ethn** [comment]
I already pushed the gem and the release for 0.3.7.  But I'm happy to do 0.3.8

**Comment by xithan** [comment]
You pushed a gem that didn't pass CI? 😟

**Comment by ethn** [comment]
At the time that I pushed it, the version itself was the only thing that
hadn't passed CI.  At that point CI was breaking everywhere, including on
master, because of the Firefox thing.

I've added a few more tweaks since, but those aren't in the release.

**Comment by ethn** [comment]
At the time that I pushed it, the version itself was the only thing that
hadn't passed CI.  At that point CI was breaking everywhere, including on
master, because of the Firefox thing.

I've added a few more tweaks since, but those aren't in the release

**Pull request merged**